### PR TITLE
config: after fix of #6558 appveyor should test changes in .gitattributes file

### PR DESCRIPTION
**Pull request by romani**
#6558

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -6,7 +6,6 @@ skip_commits:
     - .ci/
     - .circleci/
     - .github/
-    - .gitattributes
```
Comment (rnveach) [inline appveyor.yml]: @romani Do you know how this file list is used? Is it looking only at HEAD or all commits in PR?

**Comment by rnveach** [comment]
IDEA is not in CI list.

**Comment by romani** [comment]
it is always strange why some time CIs are failing to trigger.
Skip of build should still trigger build, but it takes very quick (clone only).

File: appveyor.yml (hunk 1 above)
Comment (romani) [reply appveyor.yml]: Appveyor is not quick question in execution, such CI benefits a lot from such check, but side effects are possible.

**Pull request merged**